builders/interact/ethereum-api/libraries/ethersjs.md

```diff
@@ -0,0 +1,492 @@
+---
+title: How to use Ethers.js Ethereum Library
+description: Follow this tutorial to learn how to use the Ethereum EtherJS Library to send transactions and deploy Solidity smart contracts to your Tanssi EVM ContainerChain.
+---
+
+# Ethers.js JavaScript Library
+
+## Introduction {: #introduction } 
+
+The [Ethers.js](https://docs.ethers.io/){target=_blank} library provides a set of tools to interact with Ethereum Nodes with JavaScript, similar to [Web3.js](/builders/interact/ethereum-api/libraries/web3js){target=_blank}. Tanssi EVM ContainerChains have an Ethereum-like API available that is fully compatible with Ethereum-style JSON RPC invocations. Therefore, developers can leverage this compatibility and use the Ethers.js library to interact with a Tanssi EVM ContainerChain node as if they were doing so on Ethereum. For more information on Ethers.js, check their [documentation site](https://docs.ethers.org/v5/){target=_blank}.
```

Comment:
the link at the end of this paragraph points to the v5 docs, it should point to v6

Reply:
the link at the beginning of this paragraph uses docs.ethers.io, please update all links to docs.ethers.org